packages/babel-generator/src/generators/classes.ts

```diff
@@ -114,6 +114,48 @@ export function ClassProperty(this: Printer, node: t.ClassProperty) {
   this.semicolon();
 }
 
+export function ClassAccessorProperty(
+  this: Printer,
+  node: t.ClassAccessorProperty,
+) {
+  this.printJoin(node.decorators, node);
+
+  // catch up to property key, avoid line break
+  // between member modifiers and the property key.
+  this.source("end", node.key.loc);
+
+  this.tsPrintClassMemberModifiers(node, /* isField */ true);
+
+  this.word("accessor");
```

Comment:
Nit: we should also print inner comments here, recall that in `parseClassMemberWithIsStatic` we have reset the trailing comments of `accessor` keyword to be the inner comments of accessor property:

```suggestion
  this.word("accessor");
  this.printInnerComments(node);
```

```js
class C {
  accessor /* inner comment */ [computed] = 1
}
```